hooks/playbooks/ceph-hci.yml

```diff
@@ -0,0 +1,114 @@
+- name: Load Networking Environment Definition play
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Load Networking Environment Definition
+      when: cifmw_ci_nmstate_instance_config is not defined
+      ansible.builtin.import_role:
+        name: networking_mapper
+        tasks_from: load_env_definition.yml
+
+#- name: Call Ceph deploy
+#  vars:
+#    storage_network_range: "{{ cifmw_networking_env_definition.networks.storage.network_v4 }}"
+#    storage_mgmt_network_range: "{{ cifmw_networking_env_definition.networks.storagemgmt.network_v4 }}"
+#  ansible.builtin.import_playbook: >-
+#    {{
+#      [
+#        ansible_user_dir,
+#        'src/github.com/openstack-k8s-operators/ci-framework/playbooks/ceph.yml'
+#      ]  | path_join
+#    }}
+
+- name: Deploy the Ceph secret
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  vars:
+    _ceph_role_secret_path: /tmp/k8s_ceph_secret.yml
+    _ceph_secret_path: >-
+      {{
+        [
+          cifmw_basedir,
+          'artifacts/manifests/ceph-secret.yml'
+        ] | path_join
+      }}
+    _ceph_conf_path: >-
+      {{
+        [
+          cifmw_basedir,
+          'artifacts/ceph.conf'
+        ] | path_join
+      }}
+  gather_facts: false
+  tasks:
+    - name: Deploy the Ceph secret
+      kubernetes.core.k8s:
+        kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+        api_key: "{{ cifmw_openshift_token | default(omit) }}"
+        context: "{{ cifmw_openshift_context | default(omit) }}"
+        namespace: openstack
+        src: "{{ _ceph_role_secret_path }}"
+        state: present
+
+    - name: Copy the Ceph secret to the artifacts
+      ansible.builtin.copy:
+        remote_src: true
+        src: "{{ _ceph_role_secret_path }}"
+        dest: "{{ _ceph_secret_path }}"
+        mode: "0644"
+
+    - name: Load the Ceph secret
+      ansible.builtin.slurp:
+        src: "{{ _ceph_secret_path }}"
+      register: _ceph_secret_slurp
+
+    - name: Copy the Ceph config to the artifacts
+      ansible.builtin.copy:
+        remote_src: true
+        content: >-
+          {{
+            (
+              _ceph_secret_slurp.content |
+              b64decode |
+              from_yaml
+            ).data['ceph.conf'] | b64decode
+          }}
+        dest: "{{ _ceph_conf_path }}"
+        mode: "0644"
+
+    - name: Test
```

Comment:
@pablintino 
After the ceph playbook runs there are a variety of temporary ceph files placed in /tmp [1] which you can use to skip some these extractions. The most useful of these files for this case is /tmp/ceph_client.yml [2] which you can use like this.
```
ansible.builtin.include_vars: /tmp/ceph_client.yml 
```
I'm confident the above will work since the `cifmw_ceph_client` role [3] uses this file the same way in order to create `/tmp/k8s_ceph_secret.yml`. 

[1]
```
[zuul@controller-0 ~]$ ll /tmp/ | grep ceph
-rw-r--r--. 1 zuul zuul  990 Feb  9 11:06 ceph_overrides.yml
-rw-r--r--. 1 zuul zuul  743 Feb  9 11:06 ceph_spec.yml
-rw-r--r--. 1 zuul zuul  143 Feb  9 11:06 initial_ceph.conf
-rw-r--r--. 1 zuul zuul  593 Feb  9 11:10 ceph_client.yml
-rw-------. 1 zuul zuul  305 Feb  9 11:10 ceph.client.openstack.keyring
-rw-r--r--. 1 root root  358 Feb  9 11:10 ceph.conf
-rw-------. 1 zuul zuul 1.1K Feb  9 11:10 k8s_ceph_secret.yml
[zuul@controller-0 ~]$ 
```

[2] 
```
[zuul@controller-0 tmp]$ cat ceph_client.yml 
---
cifmw_ceph_client_fsid: 2ff89baf-0f50-5019-a8fb-6b7f837f50f7
cifmw_ceph_client_cluster: ceph
cifmw_ceph_client_external_cluster_mon_ips: "[v2:172.18.0.100:3300/0,v1:172.18.0.100:6789/0],[v2:172.18.0.102:3300/0,v1:172.18.0.102:6789/0],[v2:172.18.0.101:3300/0,v1:172.18.0.101:6789/0]"
keys:
- name: client.openstack
  key: AQCUTcZlAAAAABAA2aGej+cpwqaOtmwRT0DDcQ==
  caps:
    mgr: allow *
    mon: profile rbd
    osd: profile rbd pool=vms, profile rbd pool=volumes, profile rbd pool=backups, profile rbd pool=images, profile rbd pool=cephfs.cephfs.meta, profile rbd pool=cephfs.cephfs.data
[zuul@controller-0 tmp]$ 
```
[3] https://github.com/openstack-k8s-operators/ci-framework/tree/main/roles/cifmw_ceph_client

Reply:
I will try to send a PR to extend to cifmw_ceph_client to render edpm-values-post-ceph and then you can just call that role after you have my PR from hooks/playbooks/ceph-hci.yml. I'll add you to the PR after that.

Reply:
Here's that PR https://github.com/openstack-k8s-operators/ci-framework/pull/1130